src/components/Sections/GroupedSectionsView.tsx

```diff
@@ -25,13 +26,17 @@ export const GroupedSectionView = ({
         menuState={menuState}
       />
 
-      <div className="shortcuts-list u-w-100 u-mv-3 u-mv-2-t u-mh-auto u-flex-justify-center">
+      <Grid container className="section-grid">
         {sections.map(section => (
           <GroupedSectionTile key={section.id} section={section} />
```

Comment:
Maybe add the <Grid> item here instead inside GroupedSectionTile to keep consistency with Applications as you wrap AppTitle directly in the parent component